.studiorc

```diff
@@ -53,10 +53,19 @@ export CGO_ENABLED=0
 export GO111MODULE=on
 
 # Specify where to put 'go installed' binaries
-export GOBIN=/src/bin
+setup_gobin() {
+    local vendor_hash
+    vendor_hash=$(sha256sum /src/vendor/modules.txt | cut -c1-16)
+    export GOBIN_BASE="/src/bin"
+    GOBIN="${GOBIN_BASE}/${vendor_hash}"
```

Comment:
Am I correct that should I want to, say, update the grpc vendor lib next time and I want to test on my branch then test on master, the two subdirs of /src/bin will peacefully coexist?

Reply:
Yes, but you might have to re-enter your studio or call setup_gobin when you change branches back and forth.